PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+## Please give the PR a descriptive title.
+
+## Type of change
+
+- [ ] Bug Fix
+- [ ] New Feature
+- [ ] Refactor
+- [ ] Project Build
+- [ ] Project Configuration
+
+## Submission Checklist
+
+- [ ] Have you followed the guidelines in our Contributing document?
```

Comment:
Make it [Contributing document](https://github.com/fanappics/buoy/blob/master/CONTRIBUTING.md); otherwise I have to find it

Reply:
Love it